packages/react-native/ReactAndroid/build.gradle

```diff
@@ -538,7 +538,7 @@ android {
     preBuild.dependsOn(preparePrefab)
 
     sourceSets.main {
-        res.srcDirs = ["src/main/res/devsupport", "src/main/res/shell", "src/main/res/views/modal", "src/main/res/views/uimanager"]
+        res.srcDirs = ["src/main/res/devsupport", "src/main/res/shell", "src/main/res/views/modal", "src/main/res/views/uimanager", "src/main/res/views/textinput"]
```

Comment:
Why is moving to a new resource set necessary?

Reply:
Do you mean that I could define the style in other resource sets like "src/main/res/shell/values/styles.xml"?

Reply:
> Do you mean that I could define the style in other resource sets like "src/main/res/shell/values/styles.xml"?

Yes correct. We should also consolidate them somehow but that's a topic for another PR